packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/DefaultCamera.swift

```diff
@@ -405,10 +404,8 @@ final class DefaultCamera: FLTCam, Camera {
       }
       return
     }
-    if audioWriterInput?.readyForMoreMediaData ?? false {
-      if !(audioWriterInput?.append(sampleBuffer) ?? false) {
-        reportErrorMessage("Unable to write to audio input")
-      }
+    if !(audioWriterInput?.append(sampleBuffer) ?? false) {
+      reportErrorMessage("Unable to write to audio input")
```

Comment:
i may need some help understanding this change. Can you walk me thought this change?

Reply:
That check was moved elsewhere (line 308).